Changelog.md

```diff
@@ -1,3 +1,25 @@
+**V1.13.6 - Updates**
```

Comment:
Can we squish all these down into `V1.13.0`? Makes no sense to me if the user can't select these intermediate versions

Reply:
I'll try, not sure if the CI/CD version checks will allow it... :-)

Reply:
As long as you change `Version.h` to `1.13.0` it should be ok! 🤠